modules/packages/vendored/yum.mustache

```diff
@@ -145,14 +145,14 @@ def list_updates(online):
             lastline = ""
 
         line = lastline + line
-        match = re.match("^\S+\s+\S+\s+\S+", line)
+        match = re.match(r"^\S+\s+\S+\s+\S+", line)
```

Comment:
The warning you refer to is this I assume:

```
>>> match = re.match("^\S+\s+\S+\s+\S+", line)
<stdin>:1: SyntaxWarning: invalid escape sequence '\S'
```

Yes, as explained [here](https://docs.python.org/3/library/re.html)

> Also, please note that any invalid escape sequences in Python’s usage of the backslash in string literals now generate a [SyntaxWarning](https://docs.python.org/3/library/exceptions.html#SyntaxWarning) and in the future this will become a [SyntaxError](https://docs.python.org/3/library/exceptions.html#SyntaxError)